client/src/components/Generic/Button/Button.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Button} from '@material-ui/core';
+
+function myButton(props) {
```

Comment:
would it be a good idea to validate props?

Reply:
Do I need to validate props being used by Material UI as well?